src/components/Footer/Footer.vue

```diff
@@ -0,0 +1,440 @@
+<script lang="ts" setup>
+const { messages } = useI18n();
+const year = new Date().getFullYear();
+
+const bandwidthQuoteIndex = ref<number>(-1);
+
+// @ts-ignore
+const totalBandwidthQuotes = messages.value['en-US'].footer.bandwidthRaccoonQuotes.length;
+
+function bandwidthClickHandler() {
+	if (bandwidthQuoteIndex.value + 1 >= totalBandwidthQuotes) {
+		bandwidthQuoteIndex.value = 0;
+	} else {
+		bandwidthQuoteIndex.value++;
+	}
+}
```

Comment:
Sorry, I couldn't resist trying to fix the ts-ignore and accidentally overcomplicated it.

(all you need to add to this is using `currentQuote` in the template below)

```suggestion
const { tm, t } = useI18n();
const year = new Date().getFullYear();

const bandwidthQuoteIndex = ref<number>(-1);
const quotes = computed(() => tm<string>('footer.bandwidthRaccoonQuotes') as string[]);
const totalBandwidthQuotes = computed(() => quotes.value.length);
const currentQuote = computed(() => t(`footer.bandwidthRaccoonQuotes[${Math.max(0, bandwidthQuoteIndex.value)}]`));

watch([totalBandwidthQuotes], () => {
	bandwidthQuoteIndex.value = -1;
});

function bandwidthClickHandler() {
	if (bandwidthQuoteIndex.value + 1 >= totalBandwidthQuotes.value) {
		bandwidthQuoteIndex.value = 0;
	} else {
		bandwidthQuoteIndex.value++;
	}
}
```

Reply:
oh my god. ok thanks jvs